modules/acquia_cms_site_studio/composer.json

```diff
@@ -4,8 +4,8 @@
     "description": "Handles code for Site Studio Installation & Related Configurations.",
     "license": "GPL-2.0-or-later",
     "require": {
-        "acquia/cohesion": "7.4.x-dev",
-        "acquia/cohesion-theme": "7.4.x-dev",
+        "acquia/cohesion": "~7.4",
```

Comment:
@chandan-singh7929 I think this should be `~7.4.0`, otherwise this should be same as `^7.4`. Right ?
@rajeshreeputra

Reply:
Yes, this need to be updated as `~7.4.0`.

Reply:
done